imcsdk/apis/server/inventory.py

```diff
@@ -36,23 +36,30 @@ def _extract_server_id_from_dn(handle, dn):
     return "--"
 
 
-def write_to_txt_file(temp_str):
+def _get_rack_class_id(handle):
+    from imcsdk.imcexception import ImcValidationException
+
+    if handle.platform == IMC_PLATFORM.TYPE_CLASSIC:
+        return("ComputeRackUnit")
+    elif handle.platform == IMC_PLATFORM.TYPE_MODULAR:
+        return("ComputeServerNode")
+    else:
+        raise ImcValidationException("Invalid platform type:%s detected" %
+                                     handle.platform)
+
```

Comment:
@waghswapnil although there is nothing wrong about the implementation of this method. My personal preference would be to re-write it as below. But your call.

```
class_ids = {
	IMC_PLATFORM.TYPE_CLASSIC: "ComputeRackUnit",
	IMC_PLATFORM.TYPE_MODULAR: "ComputeServerNode"
}

if handle.platform not in class_ids:
	raise ImcValidationException("Invalid platform type:%s detected" % handle.platform

return class_ids.get(handle.platform)
```